### PR TITLE
feat(@angular-devkit/build-optimizer): update source-map to 0.7

### PR DESCRIPTION
**Pull request by clydin**
source-map 0.7+ uses a WASM for increased performance.
The development version requires sync behavior capping it at 0.6.x.

Blocked on code coverage TBD refactor

### CODE REVIEW

**Comment by alexeagle** [comment]
Commit message is wrong, it's updating to 0.6 not 0.7

File: packages/angular_devkit/build_optimizer/package.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
   },
   "dependencies": {
     "loader-utils": "1.1.0",
-    "source-map": "0.5.6",
+    "source-map": "0.7.3",
```
Comment (alexeagle) [inline packages/angular_devkit/build_optimizer/package.json]: why different version from root package.json?

File: package.json [hunk 2]
```diff
@@ -95,7 +94,7 @@
     "minimist": "^1.2.0",
     "rxjs": "~6.2.0",
     "semver": "^5.3.0",
-    "source-map": "^0.5.6",
+    "source-map": "^0.6.1",
```
[the remark this answers is not in the document]
Comment (clydin) [reply package.json]: Updated the commit message.
0.6.x is the last version that is not using WASM and therefore still supports synchronous operations.
One of the development scripts ([see here](https://github.com/angular/angular-cli/blob/master/lib/istanbul-local.js)) requires synchronous support.

**Comment by hansl** [comment]
This has been marked blocked for 2 weeks. Will close this (please reopen if it should be unblocked).

**Pull request closed without merging**